build-scripts/generate_man_page.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+import argparse
+import sys
+import ssg.build_profile
+import ssg.constants
+import ssg.xml
+import os
+
+import xml.etree.ElementTree as ET
```

Comment:
we have an elementtree wrapper in the ssg module, don't import it like this, there are multiple of them and the wrapper will make sure you are using the right one